DFReader.py

```diff
@@ -363,40 +372,38 @@ def init_clock(self):
         while True:
             m = self.recv_msg()
             if m is None:
-                break;
+                break
 
             type = m.get_type()
 
-            if first_us_stamp is None:
-                first_us_stamp = getattr(m, "TimeUS", None);
-
-            if first_ms_stamp is None and (type != 'GPS' and type != 'GPS2'):
+            first_us_stamp = first_us_stamp or getattr(m, "TimeUS", None)
```

Comment:
This is a functional change.  The original check is for None, this is for truth.

Further, the separate "if" blocks are nicer in Python.  And this is coming from a Perl programmer who really likes `//=`